### PR TITLE
added "pitch" to readme

### PR DESCRIPTION
**Pull request by meiertgrootes**
Added 'pitch' section to top of readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 # evidence
 
-Word2Vec-based assisted close reading tool with support for context-based search and concept-based search.
+Doc2Vec-based assisted close reading tool with support for abtract concept-based search and context-based search.
```
Comment (jspaaks) [inline README.md]:
```suggestion
Doc2Vec-based assisted close reading tool with support for abstract concept-based search and context-based search.
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+
+## Machine supported research in humanities
```
Comment (jspaaks) [inline README.md]:
```suggestion
## Machine-supported research in humanities
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+While research in the humanities has been able to leverage the digitization of text corpora and the development of computer based text analysis tools to its benefit, the interface current systems provide the user with is incompatible with the the proven method of scholarly close reading of texts which is key in many research scenarios pursuing complex research questions.
```
Comment (jspaaks) [inline README.md]:
```suggestion
While research in the humanities has been able to leverage the digitization of text corpora and the development of computer based text analysis tools to its benefit, the interface current systems provide the user with is incompatible with the proven method of scholarly close reading of texts which is key in many research scenarios pursuing complex research questions.
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
+
+What this boils down to, is the fact that it is often restrictive and difficult to formulate selection criteria, in particular for more complex or abstract concept, in the framework of a keyword based search which is the standard entry point to digitized text collections.
```
Comment (jspaaks) [inline README.md]:
```suggestion
What this boils down to, is the fact that it is often restrictive and difficult, if not impossible, to formulate adequate selection criteria, in particular for more complex or abstract concepts, in the framework of a keyword based search which is the standard entry point to digitized text collections.
```

File: README.md [hunk 5]
```diff
@@ -1,6 +1,18 @@
...
+
+## Querying by example - close reading with tailored suggestions  
+Evidence provides an alternative, intuitive entry point into collections. Using the doc2vec framework evidence learns abstract representations of the content of the elements of the user's corpus.
```
Comment (jspaaks) [inline README.md]: One could mistake "Evidence" for its normal meaning. Can you think of a way to avoid this confusion?

Comment (jspaaks) [inline README.md]: is it word2vec or doc2vec? either way we should pick one and use it consistently throughout.

Comment (jspaaks) [inline README.md]:
```suggestion
Evidence provides an alternative, intuitive entry point into collections. Using the doc2vec framework, evidence learns abstract representations of the content of the elements of the user's corpus.
```

File: README.md [hunk 6]
```diff
@@ -1,6 +1,18 @@
...
+Departing from a set of corpus elements that the user selects as relevant starting points evidence retrieves similar elements and presents them to the user, using the users feedback to refine its retrieval.
```
Comment (jspaaks) [inline README.md]:
```suggestion
Departing from a set of corpus elements that the user selects as relevant starting points, evidence retrieves similar elements and presents them to the user, using the users feedback to refine its retrieval.
```

File: README.md [hunk 7]
```diff
@@ -1,6 +1,18 @@
...
+This enables a user to combine the power of a close-readiing approach with that of a large digitized corpus, selecting elements from the entire corpus which are likely to be of interest, but leaving the dcision up to the user.
```
Comment (jspaaks) [inline README.md]:
```suggestion
This enables a user to combine the power of a close-reading approach with that of a large digitized corpus, selecting elements from the entire corpus which are likely to be of interest, but leaving the decision up to the user.
```

**Review by jspaaks — CHANGES_REQUESTED** [review]
Nice addition, just a few typos.

File: README.md (hunk 5 above)
Comment (meiertgrootes) [reply README.md]: it is doc2vec in all cases. I'll give it a comb through

Comment (meiertgrootes) [reply README.md]: how about a different font type for the tool throughout? are there any conventions?
Or is that abhorrent to you?

Comment (meiertgrootes) [reply README.md]: @jspaaks how about a different font type for the tool throughout? are there any conventions?
Or is that abhorrent to you?

Comment (jspaaks) [reply README.md]: Whichever way you think is OK, I just wanted to raise the issue. Simplest way to resolve this is maybe just ``evidence``?

Comment (jspaaks) [reply README.md]:
```
$ grep -inr word2vec
Binary file experiments/demo/model/doc2vec.d2v matches
README.md:3:Word2Vec-based assisted close reading tool with support for context-based search and concept-based search.
README.md:50:## Generating the word2vec model
```

Comment (meiertgrootes) [reply README.md]: thanks

Comment (meiertgrootes) [reply README.md]: I was thinking the same. I'll go for that

**Comment by meiertgrootes** [comment]
I've addressed and/or committed all point raised by @jpaaks. Good to go

**Review by jspaaks — APPROVED** [review]
:hamburger:

**Comment by jspaaks** [comment]
Just went ahead and merged it

**Pull request merged**